config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-kind-periodics.yaml

```diff
@@ -31,13 +33,18 @@ periodics:
       - -c
       - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
       env:
-      # this is the key configuration
+      # Options from https://github.com/kubernetes-sigs/kind/blob/d1eecc46e30cac9d35cd32dc52677ef75ec22e18/hack/ci/e2e-k8s.sh#L79-L83
       - name: CLUSTER_LOG_FORMAT
         value: json
-      # TODO: This is a fairly arbitrary test that actually schedules some pod(s).
-      # We mostly just want cluster up/down and something that ensures the cluster is in fact up fully.
+      - name: KIND_CLUSTER_LOG_LEVEL
+        # Default is 4, but we want to exercise more log calls and get more output.
+        # TODO: go even higher, depending on log volume
+        value: "6"
+      # don't retry conformance tests
+      - name: GINKGO_TOLERATE_FLAKES
+        value: "n"
       - name: FOCUS
-        value: Job.should.run.a.job.to.completion.when.tasks.sometimes.fail.and.are.locally.restarted
+        value: \[Conformance\]|\[Driver:.csi-hostpath\]
```

Comment:
Something is broken with the focus/skip on this job, it's literally only running one test: https://testgrid.k8s.io/sig-instrumentation-tests#kind-json-logging-master

Reply:
There has been no new job run yet since this PR was merged.

Previously it did indeed only run one test.

Reply:
ah, then the previous focus was fine and this focus is broken; I missed that the previous job value was a single test

Reply:
Recent job runs executed conformance and storage tests.